reflex/vars/base.py

```diff
@@ -1017,9 +1143,131 @@ def json(self) -> str:
 OUTPUT = TypeVar("OUTPUT", bound=Var)
 
 
+class ToOperation:
+    """A var operation that converts a var to another type."""
+
+    def __getattr__(self, name: str) -> Any:
+        """Get an attribute of the var.
+
+        Args:
+            name: The name of the attribute.
+
+        Returns:
+            The attribute of the var.
+        """
+        from .object import ObjectVar
+
+        if isinstance(self, ObjectVar) and name != "_js_expr":
+            return ObjectVar.__getattr__(self, name)
+        return getattr(object.__getattribute__(self, "_original"), name)
```

Comment:
does this work here?

```suggestion
        return getattr(self._original, name)
```

i'm seeing this all over, what am i missing?

Reply:
I don't quite remember, but I had some recursion errors without it that got complex and that was once simple way of solving it. I will try without the hack to see if it works now.